README.md

```diff
@@ -227,6 +227,9 @@ TODO: Move this code over to the idseq-dag repo.
 
 ## Release notes
 
+- 3.13.1
+  - Make phylo tree and alignment viz steps more robust to missing accessions in index.
+
```

Comment:
If this goes in first, mind adding "- Bugfix in reference caching (GITHUB PR 218, JIRA IDSEQ-1759)" as 3.13.1?  When that went in, it wasn't reflected in the release history.

Reply:
Also I don't see a release note with a pipeline version for when reference caching was deployed.  Our custom is to bump the third component of the pipeline version for such infra changes -- even though they are not expected to affect results, if they do, it makes it easier to tell from the UI if you are looking at a suspect sample.   cc @kislyuk